lib/datadog/tracing/contrib/sidekiq/server_internal_tracer/redis_info.rb

```diff
@@ -13,6 +13,8 @@ def redis_info
               Datadog::Tracing.trace(Ext::SPAN_REDIS_INFO, service: configuration[:service_name]) do |span|
                 span.span_type = Datadog::Tracing::Metadata::Ext::AppTypes::TYPE_WORKER
 
+                span.set_tag(Contrib::Ext::Messaging::TAG_SYSTEM, Ext::TAG_MESSAGING_SYSTEM)
```

Comment:
Does it make sense to apply the tag when the instrumented operation isn't really related to push/pop a job from a queue?

Reply:
Hmm in this case I'm not sure what's the most correct way to go, since for example if you are using `sqs` and run the operation `ListQueues`, which is not a push/pop operation but could be considered as type `messaging` since you are doing an operation on a messaging system.

So basically this is the logic I followed to tag these operations as messaging (similar logic could be applied for `db` tags) - also I think making the decision at the integration level makes things easier for future contributors (only need to make these decisions when adding new integrations).

Not a super strong opinion though, so let me know if this makes sense to you.